recipes/quickstart/agents/Agents_101/Tool_Calling_201.ipynb

```diff
@@ -0,0 +1,774 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tool Calling 201: Llama to find Differences between two papers\n",
```

Comment:
wonder if its more comparing two papers or finding differences?

Reply:
LGTM @init27 lets merge, a follow up PR could be extending 201 or the next notebook on using OCR capabilities of 3.2.